sdks/opik_optimizer/src/opik_optimizer/meta_prompt_optimizer.py

```diff
@@ -0,0 +1,284 @@
+from typing import List, Dict, Any, Optional, Union
+import opik
+from opik.evaluation.metrics import BaseMetric
+from .base_optimizer import BaseOptimizer
+import openai
+import os
+import json
+from pydantic import BaseModel
+
+
+class OptimizationRound(BaseModel):
```

Comment:
These classes should probably be in base_optimizer.py so we can use across optimizers